src/thorlabs_cube/driver/tcube/tpz.py

```diff
@@ -183,22 +182,22 @@ async def set_pi_constants(self, prop_const, int_const):
         :param prop_const: Value of the proportional term in range [0; 255].
         :param int_const: Value of the integral term in range [0; 255].
         """
-        payload = st.pack("<HHH", 1, prop_const, int_const)
+        payload = st.pack("<HHH", _CHANNEL, prop_const, int_const)
         await self.send(Message(MGMSG.PZ_SET_PICONSTS, data=payload))
 
-    async def get_pi_constants(self):
+    async def get_pi_constants(self) -> tuple[int, int]:
         """Get the proportional and integration feedback loop constants.
 
         :return: Returns a tuple whose first element is the proportional
             term and the second element is the integral term.
         :rtype: a 2 int elements tuple : (int, int)
         """
         get_msg = await self.send_request(
-            MGMSG.PZ_REQ_PICONSTS, [MGMSG.PZ_GET_PICONSTS], 1
+            MGMSG.PZ_REQ_PICONSTS, [MGMSG.PZ_GET_PICONSTS], _CHANNEL
```

Comment:
```suggestion
            MGMSG.PZ_REQ_PICONSTS, [MGMSG.PZ_GET_PICONSTS], Tpz._CHANNEL
```
Use class constants.